app/jobs/scheduled/topic_timer_enqueuer.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Jobs
+
+  # Runs periodically to look through topic timers that are ready to execute,
+  # and enqueues their related jobs.
+  #
+  # Any leftovers will be caught in the next run, because execute_at will
+  # be < now, and topic timers that have run are deleted on completion or
+  # otherwise have their execute_at time modified.
+  class TopicTimerEnqueuer < ::Jobs::Scheduled
+    every 1.minute
+
+    def execute(args = nil)
```

Comment:
```suggestion
    def execute(_)
```

Reply:
This is not equivalent, because we want to be able to call this with no arguments. Changed it to `_args = nil` instead because I know the underscore means the var is unused.

Reply:
Ooo I didn't know we were calling this method manually :+1: